extensions/all-extensions/pom.xml

```diff
@@ -20,6 +20,11 @@
     <dependencies>
         <dependency>
             <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-ftp</artifactId>
```

Comment:
you should not be touching this file. Sockets already there... Are you sure you rebased correctly?